### ISSUE TITLE
workflows: recommend parameters and recipes

### ISSUE THREAD

**Issue by bertsky** [OCR-D/ocrd-website#104]
Sometimes a word on parameter choices would be helpful. For example, 
- `threshold` (ocrd-cis-ocropy-binarize) or `k` (ocrd-olena-binarize) parameter for binarization, 
- `maxskew` (`ocrd-cis-ocropy-deskew`) angle, 
- `find_tables` (`ocrd-tesserocr-segment-region`)
- `padding` (in `ocrd-tesserocr-segment-region`, `ocrd-tesserocr-recognize`) or `range` (in `ocrd-cis-ocropy-dewarp`)
- ...

Also, beyond full-blown workflow recommendations, simple recipes could be discussed, like:
- for deskewing, combine `ocrd-tesserocr-deskew` (which includes orientation) with `ocrd-cis-ocropy-deskew` (which is more accurate w.r.t. skew and can run afterwards)
- for binarization, combine certain noisy methods with binary denoising afterwards (like `ocrd-cis-ocropy-denoise`)
- for binarization, combine certain sensitive methods with raw denoising beforehand (like `ocrd-im6convert` with `wavelet-denoise`)
- for page segmentation, combine processors without reading order with incremental/RO-only processors (like `ocrd-cis-ocropy-segment`)
- ...

**Comment by EEngl52** [comment]
this is an important issue for sure, thanks for pointing it out @bertsky ! I suggest we will continuously amend our user guide with such pieces of information just as we gain further experience with parameter and processor choices. The pilot phase might come in quite handy in this respect

**Comment by bertsky** [comment]
@EEngl52 you mean the workflow guide, don't you?

If you make a draft PR with how and where to put such info (i.e. parameters and recipes), then I (or others) can comment/review. Or you wait for others to write about their experiences in the wiki. Maybe we should also discuss this in the VC next week. After all, we ideally want to spawn a user-oriented discussion!

**Comment by EEngl52** [comment]
I added some template pages to the wiki where such detailed recommendations can be added.

**Comment by EEngl52** [comment]
as we decided to add more in-depth recommendations to the Website Wiki: can we close this issue @bertsky ? Or do you want to keep it as a reminder for your first ideas?

**Comment by bertsky** [comment]
Yes, good idea. I'll try to add the info somewhere in the wiki, and then close here. Hopefully it gets integrated into the workflow recommendations at some point (probably after having a good working evaluation).

**Comment by kba** [comment]
@bertsky Is this adressed by the workflow-guide-from-wiki mechanism and the Notes sections there?

**Comment by kba** [comment]
Perhaps we *should* revisit the original idea of the "OCR-D cookbook" with "recipes" for common tasks and problems like the ones you mentioned. The user guide is too high-level for those, the workflow guide too low-level.

Perhaps it would make sense to combine this with the FAQ (cf. #32). We could have a docu sprint where we collect and answer common questions and based on that decide which question complexes merit a more in-depth analysis? And perhaps have those documents (FAQ and cookbook) live in the Wiki with integration into the website, like the workflows.md mechanism.
